src/app/pages/Roadmap.tsx

```diff
@@ -210,6 +211,14 @@ export const Roadmap: React.FC = (): JSX.Element => {
                     </div>
                 )}
 
+                {!loading && filteredBuckets.length === 0 &&
+                    <EmptyState
```

Comment:
Per our design pattern, this empty state component should be centered (horizontally and vertically) in its parent (the page)

Reply:
I centered the emptyState, but I did it in a hacky way because I couldn't get it to work using height 100% and flex.

Reply:
It's a little bit of a pain because of the way we have that footer configured. If you want to get it close and relatively clean:

- Only render the `<PageContent>` component around the actual contents (don't wrap the empty state component in it)
- on the empty state wrapper:
    ```
    min-height: calc(100vh - 112px);
    display: flex;
    align-items: center;
    justify-content: center;
```

Reply:
actually, your hacky way looks nicer because the footer doesn't disappear. But I would do 28vh (you're missing the h) always and not do the 15vh for small screen

Reply:
The 28vh on mobile viewports looked pretty off, I'll show a picture of what I was seeing:

Reply:
This is what an iphone 6/7/8 would look like with a vh of 20: 
![image](https://user-images.githubusercontent.com/6538289/80209870-cd420780-8600-11ea-8504-b61e189b5a88.png)

Reply:
I think 22vh on mobile is the best. 
![image](https://user-images.githubusercontent.com/6538289/80209988-f9f61f00-8600-11ea-9be1-8b9de32e04ec.png)

Reply:
It all kind of depends on how big the screen is, but this should be fine for now